eyepop/compute/status.py

```
@@ -46,58 +42,53 @@ async def wait_for_session(
     while asyncio.get_event_loop().time() < end_time:
         attempt += 1
         try:
-            log.debug(f"Health check attempt {attempt}")
-
             async with client_session.get(health_url, headers=headers) as response:
-                log.debug(f"Health check response status: {response.status}")
+                if response.status != 200:
+                    last_message = f"HTTP {response.status}"
+                    log.debug(f"GET /health - status: {response.status} (attempt {attempt})")
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)